libs/ui-toolkit/src/config/colors.ts

```diff
@@ -22,3 +22,29 @@ export const Colors = {
   VEGA_GREEN: '#26ff8a',
   VEGA_YELLOW: '#daff0d',
 };
+
+export function colorByChange(a: number, b: number) {
+  let color = Colors.WHITE;
+
+  if (a < b) {
+    color = Colors.GREEN;
+  }
+
+  if (a > b) {
+    color = Colors.RED;
+  }
+
+  return color;
+}
+
+export function colorByMarketMovement(bullish: boolean | null) {
+  if (bullish === true) {
+    return Colors.GREEN;
+  }
+
+  if (bullish === false) {
+    return Colors.RED;
+  }
+
+  return Colors.WHITE;
```

Comment:
question (non-blocking): how does this fare on a light theme? All look ok?

Reply:
this does not seem right. I will update the colors.

Reply:
reviewed with @campbellssource. the sparkline can be muted in this case. updated in my last commit.